### ISSUE TITLE
How to add multiple galleries?

### ISSUE THREAD

**Issue by hubpork** [sachinchoolur/lightGallery#149]
Thanks for the plugin.
How to add multiple galleries?
I made it like this...
```javascript
		$('.light-gallery').each( function() {
			$(this).lightGallery({
				selector : $(this).find('.gallery-thumb'),
				closable : false
			});
		})
```
Is this the correct way?
Also the browser jumpo to top... And I don't know why?
Could you help me?

**Comment by hubpork** [comment]
no browser likes the class light-gallery...
body overflow hidden causes a jump on top... and bad behaviour on IE12
Thanks

**Comment by sachinchoolur** [comment]
Hi,
Sorry for the delayed response. I have been quite busy lately..
Normally you can add multiple galleries using class selector
`$('.lightgallery').lightGallery();`

But i am not sure how your html structure is. It would be helpful if you could share your html markup..

**Comment by hubpork** [comment]
Thanks.
Did you check the .light-gallery class?

**Comment by sachinchoolur** [comment]
Hi,
I hope your issue has been solved..
> Did you check the .light-gallery class?

Could you please explain what you mean by this line?

I have released lightgallery version 1.2. 
[Here](Did you check the .light-gallery class?) you can get more information about different html markups..
